search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java

```diff
@@ -77,6 +82,29 @@ void init() throws IOException {
         outputStream = new ByteArrayOutputStream();
     }
 
+    @Test
+    void shouldSentRequestWithGivenPageSize() throws IOException {
+
+        var expectedPageSize = randomInteger();
+        InputStream request = createRequestWithPageSize(expectedPageSize);
+        handler.handleRequest(request, outputStream, context);
+        var response = GatewayResponse.fromOutputStream(outputStream, SearchResourcesResponse.class);
+        assertThat(response.getStatusCode(), is(equalTo(HTTP_OK)));
+        var actualPageSize = restHighLevelClientWrapper.getSearchRequest().source().size();
+        assertThat(actualPageSize, equalTo(expectedPageSize));
+    }
+
+    @Test
+    void shouldSentDefaultPageSizeRequestWhenPageSizeNotSubmitted() throws IOException {
+
+        InputStream request = queryWithoutQueryParameters();
```

Comment:
var

Reply:
You can replace the InputStream type with `var`